airflow/providers/databricks/hooks/databricks.py

```diff
@@ -215,6 +216,67 @@ def find_job_id_by_name(self, job_name: str) -> int | None:
         else:
             return matching_jobs[0]["job_id"]
 
+    def list_pipelines(
+        self, max_results: int = 25, pipeline_name: str | None = None, notebook_path: str | None = None
+    ) -> list[dict[str, Any]]:
+        """
+        Lists the pipelines in Databricks Delta Live Tables.
+
+        :param max_results: The limit/batch size used to retrieve pipelines.
```

Comment:
The name is confusing, I wonder if we can change it.

Reply:
Name of what exactly? Happy to change things, trying to align with how Databricks has them named.

Reply:
Ywah looks consistent with the API

Reply:
Yes it is consistent with the API, but the API uses it to determine the total number of pipelines it should return by query, but in our case we collect all the pipelines, so IMHO `max_results_by_page` or `batch_size` are more suitable. 

It's a nit, if you don't agree you can merge.

Reply:
I am happy with both

Reply:
Ah yeah I think that was a copy/paste from elsewhere. Switching it to `batch_size`. Attempting to run what tests I can locally then pushing it up.